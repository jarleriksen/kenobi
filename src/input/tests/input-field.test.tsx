import React from 'react';
import { render } from '@testing-library/react';
import { TextField } from '../';
import '@testing-library/jest-dom/extend-expect';

describe('TextField', () => {
  test('renders correctly', () => {
    const { getByTestId, asFragment } = render(<TextField />);

    const textField = getByTestId('kenobi-form-field');

    expect(textField).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
