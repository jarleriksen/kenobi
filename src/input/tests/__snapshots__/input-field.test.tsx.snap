// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-ifAKCX dZDxVL"
    data-testid="kenobi-form-field"
  >
    <input
      class="sc-bdVaJa kOtoB"
      data-testid="kenobi-input"
      id="kenobi-input-field-1"
    />
  </div>
</DocumentFragment>
`;
